test/src/gatherers/html-without-javascript.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+/* eslint-env mocha */
+
+/* NB: this is identical to the html test; it just checks afterThirdReloadPageLoad, not postProfiling */
+const HTMLWithoutJavaScriptGather = require('../../../src/gatherers/html-without-javascript');
+const assert = require('assert');
+let htmlWithoutJavaScriptGather;
+
+describe('HTML without JavaScript gatherer', () => {
+  // Reset the Gatherer before each test.
+  beforeEach(() => {
+    htmlWithoutJavaScriptGather = new HTMLWithoutJavaScriptGather();
+  });
+
+  it('returns an artifact', () => {
+    return htmlWithoutJavaScriptGather.afterThirdReloadPageLoad({
+      driver: {
+        sendCommand(cmd) {
+          switch (cmd) {
+            case 'DOM.getDocument':
+              return Promise.resolve({
+                root: {
+                  nodeId: 1
+                }
+              });
+
+            case 'DOM.querySelector':
+              return Promise.resolve({
+                nodeId: 1
+              });
+
+            case 'DOM.getOuterHTML':
+              return Promise.resolve({
+                outerHTML: '<body>Hello!</body>'
+              });
+
+            case 'Runtime.evaluate':
+              return Promise.resolve({
+                result: {value: 'Hello!'}
+              });
+
+            default:
+              throw new Error('Unsupported command');
+          }
+        }
+      }
+    }).then(_ => {
+      assert.ok(typeof htmlWithoutJavaScriptGather.artifact === 'object');
+      assert.ok(typeof htmlWithoutJavaScriptGather.artifact.html === 'string');
```

Comment:
AFAICT the audit runs against `text` not `html`. I think we should drop any references to HTML here, unless the gatherer is now gathering both at the same time (which I'd rather it didn't)